Source/Scene/Cesium3DTileFeatureTableResources.js

```diff
@@ -0,0 +1,87 @@
+/*global define*/
+define([
+        '../Core/defaultValue',
+        '../Core/defined',
+        '../Core/DeveloperError',
+        '../Renderer/WebGLConstants'
+    ], function(
+        defaultValue,
+        defined,
+        DeveloperError,
+        WebGLConstants) {
+    'use strict';
+
+    /**
+     * @private
+     */
+    function Cesium3DTileFeatureTableResources(featureTableJSON, featureTableBinary) {
+        this.json = featureTableJSON;
+        this.binaryDataView = new DataView(featureTableBinary);
+        this._cachedProperties = {};
+        this.featuresLength = 0;
+    }
+
+    Cesium3DTileFeatureTableResources.prototype.getGlobalProperty = function(semantic, componentType, count) {
```

Comment:
I think it will be cleaner to split this implementation into separate global and non-global getters.  There may be some duplication, but the logic and parameters will be decoupled.